server/src/test/java/io/crate/testing/SQLExecutor.java

```diff
@@ -228,6 +227,7 @@ public PlannerContext getPlannerContext(ClusterState clusterState, Random random
         private SessionSettingRegistry sessionSettingRegistry = new SessionSettingRegistry(Set.of(loadedRules));
 
         private Builder(ClusterService clusterService,
+                        NodeContext nodeCtx,
```

Comment:
Couldn't the builder construct the `NodeContext` itself, similar how it created the `Functions` itself before? 

Would simplify all the call-sites which use it.